addons/mail/static/src/components/discuss_sidebar/discuss_sidebar.scss

```diff
@@ -1,42 +1,7 @@
-// ------------------------------------------------------------------
-// Layout
-// ------------------------------------------------------------------
-
 .o_DiscussSidebar {
-    display: flex;
-    flex-flow: column;
-    width: $o-mail-chat-sidebar-width;
+    width: $o-mail-chat-sidebar-width; // To be paired with the values of the `searchPanel` component, still wip.
```

Comment:
No no... I'm definitely against this desire to pair them together, especially if you use the current width of control panel.

Right now the width of sidebar is 250px or 300px (big screen).

When I test the control panel (e.g. in Documents, on 1440p screen), it's 220px all screen. This is really too small.

Reply:
I agree. Moreover `searchPanel` will  not have a fixed width anymore...  a default value will be provided, but we'll let each app control its width using CSS variables.

https://github.com/odoo/odoo/pull/87687/files#diff-284c89b5c9f62ae7adc80de87d0c407db7190c5b4cc97703bae5f108273d24baR20

Reply:
It looks like I can give this conversation as resolved 😉

Reply:
Not really resolved, I think it's best not having a commit that put this comment, especially when we do not agree in desiring pairing both width values.

Reply:
> Not really resolved, I think it's best not having a commit that put this comment, especially when we do not agree in desiring pairing both width values.

Alright, I'm dropping the commit 👍

Reply:
Thanks ;-)